apis/core/lib/handlers/cube-projects.ts

```diff
@@ -0,0 +1,15 @@
+import asyncMiddleware from 'middleware-async'
+import { protectedResource } from '@hydrofoil/labyrinth/resource'
+import { shaclValidate } from '../middleware/shacl'
+import { createProject } from '../domain/cube-projects/create'
+
+export const post = protectedResource(shaclValidate, asyncMiddleware(async (req, res) => {
+  const project = await createProject({
+    resource: await req.resource(),
+    store: req.app.resources(),
+  })
+
+  res.status(201)
+  res.header('Location', project.value)
```

Comment:
Location header is necessary when doing POST so that the client knows what resource got created